src/components/organisms/UpcomingMovies/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from "react";
+
+import Slider from "../../../components/organisms/Slider";
+import Preloader from "../../atoms/Preloader";
+import ErrorMessage from "../../atoms/ErrorMessage";
+
+import { useGetUpcomingMoviesQuery } from "../../../store/api";
+
+const UpcomingMovies: FC = () => {
+  const {
+    data: upcomingMovies,
+    error: upcomingMoviesError,
+    isLoading: upcomingMoviesLoading,
+  } = useGetUpcomingMoviesQuery();
+
+  return (
+    <section>
+      {upcomingMoviesError && <ErrorMessage />}
```

Comment:
also EVERYWHERE u are showing ErrorMessage
check upcomingMoviesError error message 
there is one definitely so pelase display to user to be more specific what exactly went wrong